tests/python_functional/conftest.py

```diff
@@ -108,3 +108,19 @@ def pytest_runtest_logreport(report):
     logger = logging.getLogger("Conftest")
     if report.outcome == "failed":
         logger.error("\n{}".format(report.longrepr))
+
+
+def construct_report_file_path(item):
+    absolute_framework_dir = Path.cwd()
+    relative_report_dir = item._request.getfixturevalue("reports")
```

Comment:
I feel this a little limiting. We could allow reports to be absolute. For example with
```
def construct_report_file_path(item):
    report_dir = Path(item._request.getfixturevalue("reports"))
    testcase_name = item._request.node.name
    file_name = "testcase_{}.log".format(testcase_name)
    return Path(report_dir, testcase_name, file_name).absolute()
```

Using `.absolute()` removes the manual work for calculating `absolute_framework_dir`, and should work both with absolute and relative paths.

Reply:
+1, and fixed